public-ui/src/app/views/ehr-view/ehr-view.component.html

```diff
@@ -39,40 +39,21 @@ <h5 class="col-xs-12 box-title">
         </clr-tooltip>
       </h5>
     </div>
+
     <div class="row">
-      <div class="col-md-4 col-sm-6 col-xs-6 card-stat" >
-        {{items.length}}
-      </div>
-      <div *ngIf="searchText.value.length > 0" class="col-md-8 col-sm-6 col-xs-6 flex-xs-middle card-text">
-        matching medical concepts.
-        <clr-tooltip>
-          <clr-icon clrTooltipTrigger shape="info-standard" class="info-icon"></clr-icon>
-          <clr-tooltip-content clrPosition=right
-                             clrSize="lg" *clrIfOpen>
-            <div class="tooltip-label">Medical concepts are similar to medical terms; they capture medical information
-                in an individual’s records and may sometimes have values associated with them.
-                For example, “height” is a medical concept that has a measurement value (in centimeters).
-                These concepts are categorized into different domains. Domains are types of medical information.
-                The Data Browser searches the All of Us public data for medical concepts that
-                match the keyword or code entered in the search bar.
-                The Data Browser counts how many participants have at least
-                one mention of the matching medical concepts in their records.
-                Matching medical concepts that have the highest participant counts are returned at the top of the list.</div>
-          </clr-tooltip-content>
-        </clr-tooltip>
-      </div>
-      <div *ngIf="searchText.value.length === 0" class="col-md-8 col-sm-6 col-xs-6 flex-xs-middle card-text">
-        Showing top {{items.length}} concepts for this domain.
```

Comment:
This text and tooltip seem to still be there in the mock. Aren't they still needed?

Reply:
Changed this as per screen. Thanks.